experian-fraud-stub/Dockerfile

```diff
@@ -11,6 +11,79 @@ ENV CI3=$CI3
 ENV CI4=$CI4
 ENV CI5=$CI5
 
+ARG P1
```

Comment:
Could we pass this as a string or json blob and parse in a bash script or java

Reply:
Updated.